### PR TITLE
feat: expand relation record cards on click in Record Show page

### LINKED ISSUE #3126: Show Page - Relation field - Expand Record card
## Context

See #3099 

For this issue:
- In the Show Page, in the Relation field sections: when a card is clicked, expand it to show the related Record preview.
- When a card is expanded, the previously expanded card is collapsed.

## Figma

image

https://www.figma.com/file/xt8O9mFeLl46C5InWwoMrN/Twenty?type=design&node-id=17774-68798&mode=design&t=qpJNVAebuHYFQT5A-11

### PR DESCRIPTION
**Pull request by thaisguigon**
Closes #3126

### CODE REVIEW

**Comment by Bonapara** [comment]
@thaisguigon Great! I realized with @FelixMalfait that the feature is hard to find. Could you add a `arrow down` light icon button to make it easier to discover? (While also keeping the option to click on the entire card you implemented!) 🙏

### Figma

https://github.com/twentyhq/twenty/assets/19412894/6cbf66c8-46e0-446a-a83a-947f91506b4d

https://www.figma.com/file/xt8O9mFeLl46C5InWwoMrN/Twenty?type=design&node-id=24941-162275&mode=design&t=0ie9xJk5gwc9vGcZ-11

**Comment by thaisguigon** [comment]
> Could you add a arrow down light icon button to make it easier to discover?

@Bonapara Done!

File: packages/twenty-front/src/modules/object-record/utils/isFieldCellSupported.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+export const isFieldCellSupported = (fieldMetadataItem: FieldMetadataItem) => {
+  if (
+    [FieldMetadataType.Uuid, FieldMetadataType.Position].includes(
+      fieldMetadataItem.type,
+    )
+  ) {
+    return false;
+  }
+
+  if (fieldMetadataItem.type === FieldMetadataType.Relation) {
```
Comment (charlesBochet) [inline packages/twenty-front/src/modules/object-record/utils/isFieldCellSupported.ts:17]: we will soon be able to replace this with relationDefinitions!

**Review by charlesBochet — APPROVED** [review]
LGTM!

**Pull request merged**